app.py

```
@@ -79,18 +79,36 @@ def app_status():
+def stop_all_processes():
+    global lsl_process, app_processes
+
     # Terminate LSL process
     if lsl_process and lsl_process.poll() is None:
         lsl_process.terminate()
+        lsl_process.wait(timeout=3)
+        if lsl_process.poll() is None:
+            lsl_process.kill()
 
     # Terminate all app processes
     for app_name, process in app_processes.items():
```

Ruff (0.8.2)

Loop control variable `app_name` not used within loop body

Rename unused `app_name` to `_app_name`

(B007)